pages_/register-phone.tsx

```diff
@@ -275,7 +277,25 @@ const VerifyPhone = () => {
   );
 };
 
-const VerifyPhoneWithCaptcha = () => {
+const VerifyPhoneWithCaptcha = ({ isLoggedIn }: { isLoggedIn: boolean }) => {
+  const router = useRouter();
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    if (isLoggedIn) {
+      router.push('/');
+      setLoading(true);
```

Comment:
is the loading spinner really necessary here? if yes, shouldn't it be set before calling `router.push`?

Reply:
yes, it's, because if `loggedIn === true`, the register is being rendered for a while until the router changes the page. Regarding the order, I think it's the same.